.env.development

```diff
@@ -0,0 +1 @@
+S_PIPES_APP_TITLE=S-Pipes UI dev
```

Comment:
please change to `SPipes Editor Dev`


note, we always use only variants: 
SPipes or s-pipes but never S-Pipes

Reply:
done